client/src/pages/tasks/Show.js

```diff
@@ -293,6 +299,24 @@ const TaskShow = ({ pageDispatchers }) => {
                       }
                     />
                   )}
+                  {Settings.fields.task.childrenTasks && task.childrenTasks && (
+                    <TaskChildrenTasks
+                      dictProps={Settings.fields.task.childrenTasks}
+                      name="childrenTasks"
+                      component={FieldHelper.ReadonlyField}
+                      humanValue={
+                        <div style={{ display: "flex", flexDirection: "column" }}>
+                          {task?.childrenTasks?.map((tasks) => {
```

Comment:
Actually, task is never null/undefined here, so `task.childrenTasks?.map` would be fine. Code is also not properly formatted. And it can be simplified. The function argument should also be called `task` (singular). The `shortName` is the default LinkTo-display of a Task. So it could be:
```
                      humanValue={
                        <div
                          style={{ display: "flex", flexDirection: "column" }}
                        >
                          {task.childrenTasks?.map(task => (
                            <LinkTo
                              showIcon={false}
                              modelType="Task"
                              model={task}
                            />
                          ))}
                        </div>
                      }
```
but that is still different from childrenOrgs in pages/organizations/Show.js, which uses a ListGroup to show similar information. Probably better to be consistent (either way). If you want to use flex, there are Bootstrap classes you can use instead of hard-coded flex style rules.